crates/forge/src/scarb/config.rs

```diff
@@ -63,7 +61,7 @@ pub(crate) struct RawForkTarget {
     pub block_id: HashMap<String, String>,
 }
 
-pub(super) fn validate_raw_fork_config(raw_config: &RawForgeConfig) -> Result<()> {
+fn validate_raw_fork_config(raw_config: &RawForgeConfig) -> Result<()> {
```

Comment:
Shouldn't this return ForgeConfig? we can omit this by mistake right now

Reply:
It shouldn't, it could take the ownership of `RawForgeConfig` and return the same instance of it though